aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -2015,7 +2015,7 @@ TORCH_IMPL_FUNC(lu_unpack_out)(const Tensor& LU,
       .add_owned_input(pivots.contiguous())
       .build();
 
-    unpack_pivots_stub(pivots.device().type(), iter, std::min(m, n));
+    unpack_pivots_stub(pivots.device().type(), iter, std::min(m, n), std::max(m, n));
```

Comment:
Actually, I think that the max pivot size should be `std::min(m, n)`?

Reply:
Always min, that is correct.

Reply:
This is the max possible value to swap with, so `std::max` is correct.

Reply:
I guess it is always `n`, as it is rows that are being swapped.

Reply:
Apparently not :) 
That's what I though originally but the result of lu_factor proved me wrong and that's why I added this extra argument.

Reply:
Maybe we could insert a comment or two to clarify on that and reduce confusion?